email_totals/ses.py

```diff
@@ -299,8 +299,12 @@ def _build_tags(missing, invalid, html=False):
         descr_invalid = ('Some of the above resources have a "CostCenterOther" '
                          'tag, but do not have "CostCenter" set to "Other / 000001". ')
 
-        descr_help = ('If you need assistance modifying the '
-                      'cost center tags, please contact Sage IT.')
+        descr_help = ('To accurately track project-related costs, a cost center '
+                      'must be specified. In order to update tags, visit the '
+                      'Provisioned Products view of Service Catalog, select '
```

Comment:
hmm, the scope of this email is wider than the resources in the service catalog.   Would it make sense to move specific docs regarding the service catalog to the [service catalog wiki](https://sagebionetworks.jira.com/wiki/spaces/SC/pages/938836322/Service+Catalog+Provisioning#Update) ?

Reply:
oh, that's a great point